src/main/fc/config.c

```diff
@@ -110,10 +110,22 @@ pidProfile_t *currentPidProfile;
 #define RX_SPI_DEFAULT_PROTOCOL 0
 #endif
 
+#ifdef BOARD_HAS_VOLTAGE_DIVIDER
+#define DEFAULT_VOLTAGE_METER_FEATURES FEATURE_VBAT
+#else
+#define DEFAULT_VOLTAGE_METER_FEATURES 0
+#endif
+#ifdef BOARD_HAS_CURRENT_SENSOR
```

Comment:
BOARD_HAS_CURRENT_METER is used in multiple places
1) for features - which I want to remove anyway - in config.c
2) for setting the default type of current meter - in battery.c

we can revisit this when more cleanup is done.

today i had a bit more clarity after looking at the current state of the work and future requirements so this PR will change based on that.